meson.build

```diff
@@ -99,18 +106,31 @@ subdir('plugins')
 
 install_subdir('shaders', install_dir: 'share/wayfire')
 
-summary = [
+wlroots_xwayland_info = []
```

Comment:
A nitpick, maybe rename this to wlroots_x_info since it includes xwayland and x11 backend info.